moped-database/seeds/1602292389297_initial_seed_staging.sql

```diff
@@ -20,9 +20,10 @@ SET row_security = off;
 -- Data for Name: moped_users; Type: TABLE DATA; Schema: public; Owner: moped
 --
 
-INSERT INTO public.moped_users (first_name, last_name, title, user_id, workgroup_id, cognito_user_id, date_added, is_coa_staff, email, roles, picture, is_deleted, is_user_group_member, note) VALUES ('JD', 'Maccombs', 'Software Developer', 1, 18, NULL, '2021-03-09 17:08:14+00', true, 'jd@emailhost.xyz', '["moped-admin"]', NULL, false, false, 'JD was contracted to assist with the initial Moped research and build out.');
```

Comment:
it would probably be good to use a fake email address instead of the same email we use for a real staging login 🤷‍♂️

Reply:
👍 Yep, good call. I'll update this too. 👤